File: desktop/src/org/neggly/shootergame/desktop/DesktopLauncher.java
package org.neggly.shootergame.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import org.neggly.shootergame.ShooterGame;

public class DesktopLauncher
{
    public static void main(String[] arg)
    {
        LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
        config.width = 540;
        config.height = 960;
        config.resizable = false;
        new LwjglApplication(new ShooterGame(), config);
    }
}

File: desktop/src/org/neggly/shootergame/desktop/DesktopLauncher.kt
package org.neggly.shootergame.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import org.neggly.shootergame.ShooterGame

object DesktopLauncher
{
    @JvmStatic
    fun main(arg: Array<String>)
    {
        val config = LwjglApplicationConfiguration()
        config.title = "シューティングゲーム"
        config.width = 450
        config.height = 800
        config.resizable = false
        LwjglApplication(ShooterGame(), config)
    }
}
